tsconfig.test.json

```diff
@@ -1,4 +1,8 @@
 {
   "extends": "./tsconfig.json",
-  "include": ["lib/**/*.ts"]
-}
+  "include": ["lib/**/*.ts", "test/**/*.ts"],
+  "exclude": [
+    // TODO: fix 4.6.4; 4.7.4; 4.8.4
+    "test/xor.ts",
+  ]
```

Comment:
Please remove it when https://github.com/ts-essentials/ts-essentials/pull/413 is merged

Reply:
The PR is just merged ✅